Git-Workflow-Guidance.md

```diff
@@ -66,16 +77,18 @@ _Never push changes to your fork master, only to branches._
 * Be very careful with every pull and push command to make sure it's from/to the right repo/fork/branch and contains what is intended.  
 _Use ‘git status’ and ‘git remote -v’ frequently._
 
+* When working with CICE, treat Icepack as a separate repository.  If you need to modify both Icepack and CICE together, clone your own version of Icepack to use within CICE, create a new branch in each, and commit and push Icepack and CICE changes separately.  Do not worry about the submodule status initially.
+
 See also our [Software Development Practices](https://github.com/CICE-Consortium/About-Us/wiki/Software-Development-Practices).
 
-# Overview
+## Git Overview
 CICE consists of a top level driver and dynamical core plus the Icepack code which is included as a git submodule.  Because Icepack is a submodule of CICE, Icepack and CICE development are handled somewhat independently with respect to the github repositories even though development and testing may be done together and in the same sandbox.  
 
-Only a limited number of users are able to push to the CICE-Consortium repositories.  Users will be working on personal forks and executing pull requests to merge their fork to the CICE-Consortium repositories.  If you plan to work within the CICE-Consortium project, create a github account (https://github.com) if you don’t have one already.
+Only a limited number of users are able to push to the CICE-Consortium repositories.  Users will be working on personal forks and executing pull requests to merge their changes into the CICE-Consortium repositories.  If you plan to work within the CICE-Consortium project, create a github account (https://github.com) if you don’t have one already.
 
 A good tutorial on the overall git workflow is at [atlassian's comparing-workflows page](https://www.atlassian.com/git/tutorials/comparing-workflows).  We will be following most of the ideas discussed in those workflows.  Details about using git can be found at the git-scm site or use google search.
```

Comment:
The ["Centralized Workflow" page](https://www.atlassian.com/git/tutorials/comparing-workflows#centralized-workflow) could also be useful for the case of several developers working on the same branch in the same repo, but the "Centralized Workflow" uses only the `master` branch, whereas we would use a feature branch so I think it might just be confusing.

Reply:
If there are better ways to point to outside resources, I'm for it.